main/webapp/modules/core/scripts/views/data-table/menu-edit-cells.js

```diff
@@ -679,6 +679,16 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
 
       $('<option>').attr("value", column2.name).text(column2.name).appendTo(elmts.noteColumnSelect);
     }
+
+    var currentHeight = dialog.outerHeight();
+    var currentWidth = dialog.outerWidth();
+    dialog.resizable({
```

Comment:
Is this what causes the whole dialog to have a resize handle that seems on Firefox to allow expanding/shrinking left-to-right?
![image](https://user-images.githubusercontent.com/986438/75689347-a30a4400-5c66-11ea-8542-0c1f812698bc.png)

Reply:
not really. To facts:
1. resizing is only secondary feature requested in issue #898 to support long column names. Primary one is horizontal **scrolling** which doesn't work in FireFox at all. Only resizing works in FF, but I wasn't able to see such 'righ-side-only' stretching as your screenshot demoes. In my FF versiion (70+) resizing works properly, proportionally resizing all 3 lists and making them wider / thinner. What FF version this screenshot has been taken in ?
2. Existent html markup doesn't support vertical resizing. But in fact, vertical stretching is not required because it's about too long column names, not about column numbers.
3. Again. primary change was to add horizontal list scrolling, which was added by introducing style.overflow=auto. In reality, the only way to show full column name to user is having horizontal scrolling withing every list (not dialog window). That was added and it works fine in Chromium-based browsers as mentioned in PR description.

In fact, this PR is an attempt to provide quick remedy to the issue upon existent dialog codebase.
In Chromium, it solves the issue completely. In FireFox - only partially. Firefox is only roughly 5% share of total browsers used.
Any ideas  what platforms most of OpenRefine users are upon ? The point is, Safari @ MacOs doesn't support that horizontal scrolling for <select> tag.
It's about browser quircks as it's always been. Chrome is recommended.
Hopefully this helps ;)

Reply:
1. Yes, I have same result for FF (73.0.1) and it does needed make wider or thinner, so that's OK.
2. Thanks fine.
3. Yeap, understood.  This is definitely an improvement!  Thanks!